publication-commons/src/test/java/no/unit/nva/publication/model/business/PublishingRequestCaseTest.java

```diff
@@ -13,12 +12,16 @@
 import org.junit.jupiter.api.Test;
 
 class PublishingRequestCaseTest {
-    
+
+    public static final String FINALIZED_DATE = "finalizedDate";
+    public static final String FINALIZED_BY = "finalizedBy";
+    public static final String ASSIGNEE = "assignee";
+
     @Test
     void shouldReturnCopyWithoutInformationLoss() {
         var original = createSample(randomElement(TicketStatus.values()));
         var copy = original.copy();
-        assertThat(original, doesNotHaveEmptyValuesIgnoringFields(Set.of("assignee")));
+        assertThat(original, doesNotHaveEmptyValuesIgnoringFields(Set.of(ASSIGNEE, FINALIZED_BY, FINALIZED_DATE)));
```

Comment:
Why not test Information loss on the new fields?

Like this:

class PublishingRequestCaseTest {

    @Test
    void shouldReturnCopyWithoutInformationLoss() {
        var original = createSample(randomElement(TicketStatus.values()));
        var copy = original.copy();
        assertThat(original, doesNotHaveEmptyValues());
        assertThat(copy, is(equalTo(original)));
        assertThat(copy, is(not(sameInstance(original))));
    }
    
    private PublishingRequestCase createSample(TicketStatus status) {
        var sample = randomPublishingRequest();
        sample.setStatus(status);
        sample.setAssignee(new User(randomString()));
        sample.setFinalizedBy(new Username(randomString()));
        sample.setFinalizedDate(Instant.now());
        return sample;
    }
}

Or add them to the randomPublishingRequest() method?